rosdep/python.yaml

```diff
@@ -107,6 +107,12 @@ python-amqp:
     vivid: [python-amqp]
     wily: [python-amqp]
     xenial: [python-amqp]
+python-anyjson:
+  ubuntu:
+    trusty: [python-anyjson]
+    vivid: [python-anyjson]
+    wily: [python-anyjson]
+    xenial: [python-anyjson]
```

Comment:
This package seems to be available even in older Ubuntu distro. Therefore I think it makes sense to just list it as:

    ubuntu: [python-anyjson]